torax/tests/test_data/test_iterhybrid_predictor_corrector_tglfnn_ukaea.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2024 DeepMind Technologies Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Identical to test_iterhybrid_predictor_corrector but with TGLFNNukaea
+transport"""
+from copy import deepcopy
+from pathlib import Path
+
+from torax.tests.test_data import test_iterhybrid_predictor_corrector
+
+base_path = Path(
+    '/home/theo/documents/ukaea/tglfnn-ukaea/MultiMachineHyper_1Aug25'
+)
```

Comment:
What is the best way of loading the weights in the CI? The repo is https://github.com/ukaea/tglfnn-ukaea.